renku/ui/cli/run.py

```diff
@@ -132,6 +132,14 @@
     This only affects files and directories; command options and flags are
     still treated as inputs.
 
+.. topic:: Disabling parameter detection (``--no-parameter-detection``)
+
+    Inputs that aren't files or directories are automatically detected as
+    parameters in ``renku run``. You can disable this feature by passing the
+    ``--no-parameter-detection`` flag. You can still manually specify
+    parameters using ``--param`` arguments mentioned above or using the
+    ``renku.api.Parameter`` class in Python code.
```

Comment:
Maybe we should very briefly mention what happens to the part of a command passed to renku run that is not a file or directory and the `--no-parameter-detection` flag is used? I am assuming then this part of the command is fully ignored? Or more like it is recorded in the metadata but then it cannot be dynamically modified with rerun/run?